app/src/pages/nanomine/contactus/contactus.js

```diff
@@ -77,29 +76,31 @@ export default {
         return
       }
       try {
-        await this.$apollo.mutate({
-          mutation: CONTACT_US_QUERY,
-          variables: {
-            input: {
-              fullName: this.name,
-              email: this.email,
-              purpose: this.contactType,
-              message: this.message
-            }
-          }
-        })
+        const payload = {
+          fullName: this.name,
+          email: this.email,
+          purpose: this.contactType,
+          message: this.message
+        }
+        const query = await this.$store.dispatch('contact/contactUs', payload)
+        if (!query) {
+          throw new Error('Failed to Submit')
+        }
         this.openDialogBox({
           type: 'Success',
           header: 'Submitted successfully',
           content: 'We would get back to you shortly'
```

Comment:
I know this isn't part of this PR, but noticing a typo here: should probably be "We will get back to you shortly" instead of "would"